ember/app/controllers/posts/base.js

```diff
@@ -27,8 +27,6 @@ export default Ember.Controller.extend({
 
       if(this.get("model.isPublished") === true){
         this.transitionToRoute("posts.show", this.get("model.postSlug"));
-      } else {
-        this.transitionToRoute("posts.edit", this.get("model.postSlug"));
```

Comment:
I should address this.

- [ ] TODO: Either add this back or add auto save to the "new" route